boards.txt

```diff
@@ -873,38 +1014,47 @@ adafruit_metro_m4.menu.debug.on=On
 adafruit_metro_m4.menu.debug.on.build.flags.debug=-g
 
 
+# -----------------------------------
 # Adafruit Grand Central M4 (SAMD51)
-# ------------------------------
+# -----------------------------------
 adafruit_grandcentral_m4.name=Adafruit Grand Central M4 (SAMD51)
+
+# VID/PID for Bootloader, Arduino & CircuitPython
 adafruit_grandcentral_m4.vid.0=0x239A
 adafruit_grandcentral_m4.pid.0=0x8031
 adafruit_grandcentral_m4.vid.1=0x239A
 adafruit_grandcentral_m4.pid.1=0x0031
-adafruit_grandcentral_m4.vid.1=0x239A
-adafruit_grandcentral_m4.pid.1=0x0032
-adafruit_grandcentral_m4.upload.tool=bossac18
+adafruit_grandcentral_m4.vid.2=0x239A
```

Comment:
one of the common typo with duplication of `vid.1` instead of `vid.1 and vid.2`

Reply:
thanks!